# Hypothesis Testing
In hypothesis testing, the purpose is to determine the probability of an event occuring, given population parameters. 

Example: 
A 2015 study is conducted which concludes that students watch an average of 26 hours per week of TV in 2015, with a stddev of 10.

Now, in 2016, after taking a random sample of 50 students, the results of this sample show a mean of 24 hours. 

The goal is to determine how probable it is that the number of hours watched by students has decreased, or put another way, 'What is the probability that this sample could be taken from the original dataset?'

**H0:** The number of hours of TV per week watched by students has not decreased. 

**H1:** The number of hours of TV per wee watched by students has increased. 


**Strategy**
Treat the problem as a CLT problem. 
1. Create sample mean distribution
    mean = population mean 
    sd = population sd / root(n)
    
2. Compute distance from mean in terms of sd
    z = (event - sample mean) / sample sd
   
3. Lookup in Z-Table




In [1]:
import math

#Step 1
populationmean = 26
populationsigma = 10

samplesize = 50
samplemean = populationmean 
samplesigma = 10 / math.sqrt(samplesize)

#Step 2 
event = 24
distance = 24 - samplemean 
z = distance / samplesigma
z

-1.4142135623730951

Step 3 
Z table @ -1.41 =  0.081 = 8.1%

8.1% means there's an 8.1% chance that the event of picking a sample with mean of 24 from our population could occur. 

8.1% is quite high, and therefore the event seems too likely to reject the null hypothesis. **The null hypothesis is accepted**

A small probability would mean, it's very unlikely that one could obtain the event, given that the null hypothesis is true. Hence, the null hypothesis would have to be rejected. 

# Hypothesis Testing: Rejection Regions

<img src='http://www.z-table.com/uploads/2/1/7/9/21795380/7807141_orig.png' style='width: 50vw; margin-left: 0;'/>
Similarly to looking up a value in terms of standard deviation in the Z table. *event-samplemean / samplestddev*

We can also lookup the standard deviation value that corresponds to a given probability. 

#### Example
Tell me at what the value is there a 10% or less probability of occurence?

1. Lookup the value in the Z table cell. 
2. Find the column and rows that correspond to the porbability you looked up.

The value you come away with (column, row) is called the 'Z-critical value'. This is 'how many standard deviations from the mean'.

The value you're looking up, the probability, is called the p-critical value. 

The **rejection region** can be interpreted as, 'The probability of randomly obtaining a result from this this area, is equal to the p-critical value'.